corehq/apps/export/static/export/js/export_list.js

```diff
@@ -158,12 +158,14 @@ hqDefine("export/js/export_list", [
     };
 
     var exportListModel = function (options) {
-        assertProperties.assert(options, ['isDeid', 'modelType', 'urls']);
+        assertProperties.assert(options, ['isDailySavedExport', 'isDeid', 'isFeed', 'modelType', 'urls']);
```

Comment:
Is this same JS model used for both Daily Saved Exports and regular ones?  Are they the same pages?  The appearance and functionality is pretty different.  Certainly makes sense to share pagination logic however, just surprised that this is used for both.

Reply:
Yeah, the daily saved / dashboard feeds UIs are supersets of the basic export page: same basic structure, but daily saved adds the update data button & progress bar to the first column and also additional columns for enable/disable and editing filters.

I think it might be an improvement to redesign the daily saved & dashboard feed pages so that they're *not* just extensions of base exports, because there's just so much crammed into that first column (progress bar, etc) but that's a bigger question for the future.